src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -286,10 +286,12 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
   confirmArticles = false
   isEditingType = false
   dropdown: boolean = null
+  supportedEntityTypes = null
 
   /** Called when component is mounted. */
   mounted (): void {
     this.initializeEntityType()
+    this.supportedEntityTypes = GetFeatureFlag('supported-alteration-change-business-types') || []
```

Comment:
The feature flags can update in real time, so you might want to fetch them in `entityTypeOptions()` instead.

If this component is only mounted when needed (ie, short lifespan) then this probably doesn't matter, but if this component is always mounted then you should check the FF right before using it.

Reply:
I didn't want to be slamming it too much, rather do it on mount - this was intentional

Reply:
OK, was just checking that you thought about it.

BTW, getting the feature flag here doesn't cause a network message (which is why it isn't async). The FFs are automatically synced in the background via a stream, I think.